fotoapparat/src/main/java/io/fotoapparat/parameter/range/ContinuousRange.java

```diff
@@ -0,0 +1,32 @@
+package io.fotoapparat.parameter.range;
+
+import android.os.Build;
+import android.support.annotation.RequiresApi;
+
+/**
+ * Created by ychen on 5/25/2017.
+ */
+
+@RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
```

Comment:
We would like to make this class available to all platforms. We can achieve that by not having `android.util.Range` as a field of the class and instead really implementing range ourselves.